Actions/BuildPowerPlatform/BuildPowerPlatform.ps1

```diff
@@ -1,9 +1,13 @@
 [CmdletBinding()]
 param(
-    [Parameter(Position = 0, mandatory = $true)] [string] $CompanyId,
-    [Parameter(Position = 1, mandatory = $true)] [string] $EnvironmentName,
-    [Parameter(Position = 2, mandatory = $true)] [string] $SolutionFolder
+    [Parameter(Position = 0, mandatory = $true)] [string] $solutionFolder,
+    [Parameter(Position = 1, mandatory = $false)] [string] $CompanyId,
+    [Parameter(Position = 2, mandatory = $false)] [string] $EnvironmentName,
+    [Parameter(Position = 3, mandatory = $false)] [string] $appBuild,
+    [Parameter(Position = 4, mandatory = $false)] [string] $appRevision,
+    [Parameter(Position = 5, mandatory = $false)] [string] $managed
 )
+$ErrorActionPreference = "Stop"; $ProgressPreference = "SilentlyContinue"; Set-StrictMode -Version 2.0
```

Comment:
isn't this set in action.yaml ?

Reply:
Yes, but there was a disconnect from running the tests and running the actions. (i.e. some issue only show up in actions). 

We could also set this in the test if this gives to much duplication.

Reply:
Yes, it shouldn't be here - it should be set in tests if needed.
Probably in BeforeAll where the .ps1 file is included.